Formula/doxygen.rb

```diff
@@ -23,7 +23,7 @@ class Doxygen < Formula
   depends_on "cmake" => :build
   depends_on "graphviz" => :optional
   depends_on "qt5" => :optional
-  depends_on "llvm" => "with-clang" if build.with? "libclang"
+  depends_on "llvm" if build.with? "libclang"
```

Comment:
Now that we've finally got a green build: any objections to doing that when we pull?